scipy/linalg/_basic.py

```diff
@@ -68,31 +68,31 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
         Square input data
     b : (N, NRHS) array_like
         Input data for the right hand side.
-    sym_pos : bool, optional, deprecated
+    sym_pos : bool, default: False, deprecated
         Assume `a` is symmetric and positive definite.
 
         .. deprecated:: 0.19.0
             This keyword is deprecated and should be replaced by using
            ``assume_a = 'pos'``. `sym_pos` will be removed in SciPy 1.11.0.
 
-    lower : bool, optional
-        If True, only the data contained in the lower triangle of `a`. Default
-        is to use upper triangle. (ignored for ``'gen'``)
-    overwrite_a : bool, optional
+    lower : bool, default: False
+        If True, the calculation uses only the data in the lower triangle of
+        `a`; entries above the diagonal are ignored. If False (default), the
+        calculation uses only the data in the upper triangle of `a`; entries
+        below the diagonal are ignored. Ignored if ``assume_a == 'gen'``.
+    overwrite_a : bool, default: False
         Allow overwriting data in `a` (may enhance performance).
-        Default is False.
-    overwrite_b : bool, optional
+    overwrite_b : bool, default: False
         Allow overwriting data in `b` (may enhance performance).
-        Default is False.
-    check_finite : bool, optional
+    check_finite : bool, default: True
         Whether to check that the input matrices contain only finite numbers.
         Disabling may give a performance gain, but may result in problems
         (crashes, non-termination) if the inputs do contain infinities or NaNs.
-    assume_a : str, optional
+    assume_a : str, {'gen', 'sym', 'her', 'pos'}
```

Comment:
```suggestion
    assume_a : str, {'gen', 'sym', 'her', 'pos'}, default: 'gen'
```
Given all the other arguments state the default it would be nice to do the same here for consistency

Reply:
From the [style guide]( https://numpydoc.readthedocs.io/en/latest/format.html)
> When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first.

The example does not also call out the default explicitly, so this follows the example.

Reply:
Thanks didn't realise this was the case; you learn something new everyday!